.github/workflows/ci.yml

```diff
@@ -158,15 +230,70 @@ jobs:
         cd allennlp-models
         pip install --upgrade --upgrade-strategy eager -e . -r dev-requirements.txt
 
-    - name: Run models tests
+    - name: Run tests
       run: |
-        cd allennlp-models && make test
+        cd allennlp-models
+        make test-with-cov COV=allennlp
+        mkdir coverage
+        mv coverage.xml coverage/
+
+    - name: Save coverage report
+      if: matrix.python == '3.7' && github.repository == 'allenai/allennlp' && (github.event_name == 'push' || github.event_name == 'pull_request')
```

Comment:
Why do we fix 3.7 here?

Reply:
Oops, this was needed when these tests ran on multiple versions of Python, because we only needed one coverage report from this job.